blivet/osinstall.py

```diff
@@ -239,11 +239,11 @@ def _findExistingInstallations(devicetree):
             device.format.mount(options=options, mountpoint=getSysroot())
         except Exception: # pylint: disable=broad-except
             log_exception_info(log.warning, "mount of %s as %s failed", [device.name, device.format.type])
-            device.format.umount(mountpoint=getSysroot())
```

Comment:
I think device.format.unmount is the right thing to call here, isn't it? Looking at blivet/formats/fs.py the unmount method does several things, including udev settling, calling util.umount and running lsof if it can't be unmounted.